packages/labs/ssr/src/lib/server-template.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {
+  html as baseHtml,
+  type CompiledTemplateResult,
+  type TemplateResult,
+} from 'lit-html';
+
+/**
+ * A lit-html template that can only be rendered on the server, and cannot be
+ * hydrated.
+ *
+ * This is a performance optimization, causing the server not to emit comment
+ * markers for updating the dynamic parts of the server-rendered DOM, and
+ * supports rendering into raw text elements like <title> and <textarea>.
```

Comment:
Leading with `performance optimization` may make people try and prefer this over the normal template, which can become an issue if they try and nest the server template in a client template.

I think we should lead with the features.

Reply:
Good point. I dropped the docs on the module-local `SERVER_ONLY` constant, and used the description from the README to document the `serverhtml` function.